edx_proctoring/static/proctoring/js/views/proctored_exam_info.js

```diff
@@ -0,0 +1,158 @@
+                'border-top': '5px solid ' + color,
+                'margin-bottom': '15px'
+            });
+
+            $el.find('.onboarding-status').css({
+                'font-weight': 'bold',
+                'margin-bottom': '15px'
+            });
+
+            $el.find('.onboarding-status-message').css({
+                'margin-bottom': '15px'
+            });
+
+            $el.find('.action').css({
+                display: 'block',
+                'font-weight': '600',
+                'text-align': 'center',
+                'text-decoration': 'none',
+                padding: '15px 20px',
+                border: 'none'
+            });
+
+            $el.find('.action-onboarding').css({
+                color: '#ffffff',
+                background: '#98050e',
+                'margin-bottom': '15px'
+            });
+
+            $el.find('.action-info-link').css({
+                border: '1px solid #0d4e6c'
+            });
+        },
+
+        getExamAttemptText: function(status) {
+            if (status in examStatusReadableFormat) {
+                return examStatusReadableFormat[status];
+            } else {
+                return {status: status || 'Not Started', message: ''};
+            }
+        },
+
+        shouldShowExamLink: function(status) {
+            // show the exam link if the user should retry onboarding, or if they haven't submitted the exam
+            var NO_SHOW_STATES = ['submitted', 'second_review_required', 'verified'];
+            return !NO_SHOW_STATES.includes(status);
+        },
+
+        render: function() {
+            var statusText = {};
+            var data = this.model.toJSON();
+            // only update html if something is returned from the endpoint, i.e. the course has an
+            // onboarding exam
+            if (this.template && data) {
+                this.status = data.onboarding_status;
+                statusText = this.getExamAttemptText(data.onboarding_status);
+                data = {
+                    onboardingStatus: statusText.status,
+                    onboardingMessage: statusText.message,
+                    showOnboardingReminder: data.onboarding_status !== 'verified',
+                    showOnboardingExamLink: this.shouldShowExamLink(data.onboarding_status),
+                    onboardingLink: data.onboarding_link
+                };
+
+                $(this.el).html(this.template(data));
+            }
+        },
+
+        loadTemplateData: function() {
+            var self = this;
+            // only load data/render if course_id is defined
+            if (self.course_id) {
+                $.ajax({url: self.template_url, dataType: 'html'})
+                    .done(function(templateData) {
+                        self.template = _.template(templateData);
+                        self.hydrate();
+                    });
+            }
+        },
+
+        hydrate: function() {
```

Comment:
#NIT This is a creative function name. But wouldn't it be more appropriate to be called sync_with_server or something like that?

Reply:
I think this is standard for Backbone!

Reply:
Huh... learning new things today.